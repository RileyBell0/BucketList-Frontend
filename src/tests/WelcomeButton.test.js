import renderer from "react-test-renderer";
import Welcome_Button from "../components/WelcomeButton";

it("Matches previous snapshot", () => {
  const comp = renderer.create(
    <Welcome_Button id={"test_id"} value={"test_value"} />
  );
  let tree = comp.toJSON();
  expect(tree).toMatchSnapshot();
});
