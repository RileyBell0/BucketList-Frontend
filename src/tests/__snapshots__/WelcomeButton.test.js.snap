// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches previous snapshot 1`] = `
<p
  className="welcome-button undefined"
/>
`;
